code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -98,107 +95,92 @@
 /obj/item/weapon/reagent_containers/glass/bottle/capsaicin
 	name = "Capsaicin Bottle"
 	desc = "A small bottle. Contains hot sauce."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle3"
+	icon_state = "round_bottle"
 	list_reagents = list("capsaicin" = 30)
 
 /obj/item/weapon/reagent_containers/glass/bottle/frostoil
 	name = "Frost Oil Bottle"
 	desc = "A small bottle. Contains cold sauce."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle17"
+	icon_state = "round_bottle"
 	list_reagents = list("frostoil" = 30)
 
 /obj/item/weapon/reagent_containers/glass/bottle/morphine
 	name = "Morphine Bottle"
 	desc = "A small bottle. Contains morphine."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle17"
+	icon_state = "round_bottle"
 	list_reagents = list("morphine" = 30)
 
 /obj/item/weapon/reagent_containers/glass/bottle/ether
 	name = "Ether Bottle"
 	desc = "A small bottle. Contains ether."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle17"
+	icon_state = "round_bottle"
 	list_reagents = list("ether" = 30)
 
 /obj/item/weapon/reagent_containers/glass/bottle/charcoal
 	name = "Charcoal Bottle"
 	desc = "A small bottle. Contains charcoal."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle17"
+	icon_state = "round_bottle"
 	list_reagents = list("charcoal" = 30)
 
 /obj/item/weapon/reagent_containers/glass/bottle/epinephrine
 	name = "Epinephrine Bottle"
 	desc = "A small bottle. Contains epinephrine."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle17"
+	icon_state = "round_bottle"
 	list_reagents = list("epinephrine" = 30)
 
 /obj/item/weapon/reagent_containers/glass/bottle/pancuronium
 	name = "Pancuronium Bottle"
 	desc = "A small bottle of pancuronium."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle14"
+	icon_state = "round_bottle"
 	list_reagents = list("pancuronium" = 30)
 
 /obj/item/weapon/reagent_containers/glass/bottle/sulfonal
 	name = "Sulfonal Bottle"
 	desc = "A small bottle of Sulfonal."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle19"
+	icon_state = "round_bottle"
 	list_reagents = list("sulfonal" = 30)
 
 //Reagent bottles
 
 /obj/item/weapon/reagent_containers/glass/bottle/reagent
 	name = "Reagent Bottle"
 	desc = "A bottle for storing reagents"
-	icon_state = "rbottle"
+	icon_state = "reagent_bottle"
 	volume = 50
 
 /obj/item/weapon/reagent_containers/glass/bottle/reagent/oil
 	name = "Oil Bottle"
 	desc = "A reagent bottle. Contains oil."
-	icon_state = "rbottle1"
```

Comment:
Should be resolved now, thanks for catching that
![bottlesstackingproperly](https://cloud.githubusercontent.com/assets/12377767/18221450/6e1aa92e-714b-11e6-8d88-f1755b6f42df.PNG)